bundle/manifests/node-healthcheck-operator.clusterserviceversion.yaml

```diff
@@ -45,14 +45,13 @@ metadata:
     categories: OpenShift Optional
     containerImage: quay.io/medik8s/node-healthcheck-operator:latest
     createdAt: ""
-    description: An operator to monitor node healthcheck and remediate via 3rd party
-      providers like Self Node Remediation.
+    description: Detect failed Nodes and trigger remediation with e.g. Self Node Remediation.
     olm.skipRange: '>=0.2.0 <0.3.0'
     operators.operatorframework.io/builder: operator-sdk-v1.18.0+git
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
     repository: https://github.com/medik8s/node-healthcheck-operator
     support: Medik8s
-  name: node-healthcheck-operator.v0.0.1
+  name: node-healthcheck-operator.v0.0.3
```

Comment:
Isn't it supposed to stay `0.0.1` after running `make bundle`?

Reply:
yes, fixed